tests/modules/programs/rofi/wofi-assert-on-plugins.nix

```
@@ -0,0 +1,26 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let dummyPackage = x: pkgs.runCommandLocal "dummy-${x}" { } "";
```

```suggestion
let dummyPackage = x: pkgs.runCommandLocal "dummy-${x}" { } "mkdir $out";
```

This should resolve the failure in the tests.

---

This will resolve previous failures, but wont trigger assertions.